moto/ec2/models.py

```diff
@@ -5562,6 +5562,30 @@ def modify_spot_fleet_request(
         return True
 
 
+class SpotPrice(object):
+    def describe_spot_price_history(self, instance_types=[], filters=None):
+        matches = INSTANCE_TYPE_OFFERINGS["availability-zone"]
+        matches = matches[self.region_name]
+
+        def matches_filters(offering, filters):
+            def matches_filter(key, values):
+                if key == "availability-zone":
+                    return offering.get("Location") in values
+                elif key == "instance-type":
+                    return offering.get("InstanceType") in values
+                else:
+                    return False
+
+            return all([matches_filter(key, values) for key, values in filters.items()])
+
+        matches = [o for o in matches if matches_filters(o, filters)]
+
+        if instance_types:
```

Comment:
Can we get a test for this as well, to verify that a call to `describe_spot_price_history(InstanceTypes=[..])` works?